18/election_transaction.yaml

```diff
@@ -0,0 +1,35 @@
+---
+"$schema": "http://json-schema.org/draft-04/schema#"
+type: object
+title: Election Schema
+required:
+- operation
+- outputs
+properties:
+  asset:
```

Comment:
`asset` has not been placed under `required`. Do we expect to have an election without a proposal?

Reply:
If we do not have a reason to require it, why require?

Reply:
Then can we remove the asset field altogether?